src/components/VideoList/VideoList.component.jsx

```diff
@@ -4,11 +4,11 @@ import VideoItem from '../VideoItem';
 import { GridContainer } from './VideoList.styles';
 
 const VideoList = ({ videos }) => {
-  const renderedVideos = videos.map(({ etag, snippet }) => {
-    return <VideoItem key={etag} video={snippet} />;
+  const renderedVideos = videos.map((video) => {
+    return <VideoItem key={video.id.videoId} video={video} videos={videos} />;
   });
```

Comment:
In order to avoid this navigation through video object, you can simply use the index provided by map as the component key value:
```suggestion
  const renderedVideos = videos.map((video, index) => <VideoItem key={index} video={video} videos={videos} />;
```